packages/base/src/util/PopupUtils.ts

```diff
@@ -72,12 +71,12 @@ const isClickInRect = (event: MouseEvent | TouchEvent, rect: DOMRect) => {
 };
 
 interface PopupInterface { // Refactor: replace with Popup.js
-	showAt: (opener: DOMReference, preventInitialFocus: boolean) => Promise<void>,
+	_show: () => void,
```

Comment:
maybe we can replace that interface with Popup. I think that it was introduced for instanceOfPopup method until popup migration.

Reply:
I did think about it but saw that Popup is in webc.main, while this these utils are in webc.base. Should we move the Popup type? It could make some sense.

Reply:
If it's used as a type there is no problem from which package it will be used because after compilation to JS it will be removed.

Reply:
We can address this seperately